symbols/arm9.yml

```
@@ -4141,10 +4318,21 @@ arm9:
         NA: 0x2025230
         JP: 0x20252B8
       description: |-
-        Note: unverified, ported from Irdkwia's notes
+        A special version of strcpy for handling names. Appears to use character 0x7E as some kind of
+        formatting character in NA?
+        
+        r0: dst
+        r1: src
+    - name: SpecialNStrcpy
+      address:
+        NA: 0x2025314
+      description: |-
+        A special version of strcpy for handling names. Appears to use character 0x7E as some kind of
```

Should this be:

```suggestion
        A special version of strncpy for handling names. Appears to use character 0x7E as some kind of
```

?